src/player.cpp

```diff
@@ -10098,7 +10147,7 @@ void player::consume_effects(item *eaten, const it_comest *comest, bool rotten)
     mod_stomach_food(nutrition_for(comest) * factor * hunger_factor);
     mod_stomach_water(comest->quench * factor);
     if (unhealthy_allowed || comest->healthy > 0) {
-        mod_healthy_mod(comest->healthy);
+        mod_healthy_mod(comest->healthy, 5 * comest->healthy);
```

Comment:
`5 * comest->healthy` is really low. For vast majority of foods, it means that they won't go above the value supplied by vitamins.

Should be hardcoded to 200, 100 or the like. But above 50, not to make vitamins mandatory.